FileMapper/blockerFunction.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Decorate function to make it blocking
+ * Emulate blocking behavior by manually blocking the execution by using
+ * utilities.sleep, If the mapping is not found, the function will try 5 times
+ * with 10 seconds interval between attempts. If the mapping is still not found,
+ * error will be thrown.
+ *
+ * @param {function} functionToBeBlocked Function that needs to be blocking
+ * @return {function} Decorated blocking function
+ */
+function blockerFunction(functionToBeBlocked) {
```

Comment:
Should we reuse this here? -> https://github.com/googleinterns/filesystem-in-appsscript/pull/71/files FileSystem/Util.js method instead of a copy?

Reply:
Actually, i was also thinking about this but for file mapper APIs we need to load the config data too before every repeated execution. Can't think of a way to do this, can you suggest some way.

Reply:
You can do that in the functionToBeBlocked (first statement there?) Or we can extend the method there to take an additional method to be called as a precursor to the main method?

Reply:
Doing that in functionToBeBlocked will load the config even at times when it is not needed to be loaded, instead your other suggestion seems good, we can do that. I will work on that part then.